packages/diagram/src/module/base/diagramModule.ts

```diff
@@ -45,6 +42,40 @@ const styleValueType = namedType(
     "unset | var | calc"
 );
 
+/**
+ * Type for a selector
+ */
+const selectorType = objectType(
+    new Map([
+        ["selectorType", stringType],
+        ["selectorValue", stringType]
+    ])
+);
+
+/**
+ * Type for a font object
+ */
+const fontType = objectType(
+    new Map([
+        ["url", stringType],
+        ["name", optional(stringType)],
+        ["variationSettings", optional(objectType())]
+    ])
+);
+
+/**
+ * Type for a font family object
+ */
+const fontFamilyType = objectType(
+    new Map([
+        ["fontFamily", stringType],
+        ["normal", fontType],
+        ["italic", fontType],
+        ["bold", fontType],
+        ["boldItalic", fontType]
```

Comment:
for now, yes
if we want, we can change that behavior (however than in a future PR as this will also affect how subsetting works)
> So in case one of them is unsupported, the normal URL should be used again?
Or what should be done in this case?

yes, or the URL of the italic version of another font you like